test/python/test_axpy_pipelined.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import pytest
+import getpass
+import tensorflow as tf
+import ngraph_bridge
+
+import numpy as np
+from common import NgraphTest
+
+# This test needs access to axpy_pipelined.py script
+# present in ngraph-bridge/examples
+# so this path needs to be added to python path when running this
+
+# For eg. when running the test from ngraph-bridge/build_cmake/test/python
+# you can add this path as below
+#sys.path.insert(0, '../../examples')
+
+from axpy_pipelined import *
+
+
+class TestAxpyPipelined(NgraphTest):
+
+    def test_axpy_pipelined(self):
+        input_array, output_array, expected_output_array = run_axpy_pipeline()
```

Comment:
set and unset the "NGRAPH_TF_PREFETCH" flag, you can refer to this example
https://github.com/tensorflow/ngraph-bridge/blob/master/test/python/test_variableops_static_input.py#L113